frontend/js/player-adapter-html5.js

```diff
@@ -85,7 +85,7 @@ define(
 
                 window.Hls = Hls;
                 mediaElementPlayer = new mejs.MediaElementPlayer(targetElement, {
-                    renderers: ["html5", "native_hls"],
+                    renderers: ["native_hls", "html5"],
```

Comment:
Is this necessary even with the correct sort order of the videos? 🤔

Looking at the docs for this setting I also wonder: Can we get away with not specifying this at all?